src/UI Component/Login/signuppage.js

```diff
@@ -0,0 +1,194 @@
+import React, { useState, useEffect } from "react";
+import SignIn from "../Login/login"
+
+
+const SignUpPage = () => {
+  const [formData, setFormData] = useState({
+    firstname: "",
+    lastname: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+  });
+
+  const [errors, setErrors] = useState({});
+  const [isSignUpSuccess, setIsSignUpSuccess] = useState(false);
+  const [showLogin, setShowLogin] = useState(false);
+
+  useEffect(() => {
```

Comment:
This is a good approach but you don't have to do this here. You can easily clear the state where  they submit the form.